src/components/Select/PSelectButton.vue

```diff
@@ -34,6 +34,10 @@
   const el = computed(() => buttonElement.value)
 
   defineExpose({ el, wrapper })
+
+  const props = defineProps<{
+    small?: boolean,
```

Comment:
This is because your defining `const props` but not using `props.small` in the script. And in the template you shouldn't use `props.small` just `small`